ambari-server/src/main/java/org/apache/ambari/server/state/ServiceInfo.java

```diff
@@ -503,6 +517,9 @@ public ComponentInfo getComponentByName(String componentName){
     return null;
   }
   public boolean isClientOnlyService() {
+    if (category.equals(ServiceCategory.CLIENT)) {
```

Comment:
Can `category` be `null`?

Reply:
Will change to ServiceCategory.CLIENT.equals(category)